File: src/main/java/org/dbtools/query/shared/QueryUtil.java
/*
 * QueryUtil.java
 *
 * Copyright 2008 Jeff Campbell. All rights reserved. Unauthorized reproduction 
 * is a violation of applicable law. This material contains certain 
 * confidential or proprietary information and trade secrets of Jeff Campbell.
 */
package org.dbtools.query.shared;

/**
 * @author Jeff
 */
public final class QueryUtil {


    private QueryUtil() {}

    public static String formatLikeClause(String column, String value) {
        return column + " LIKE " + value;
    }

    public static String formatIgnoreCaseLikeClause(String column, String value) {
        return formatLikeClause(column, value);
    }
}

File: dbtools-query/src/commonMain/kotlin/org/dbtools/query/shared/util/QueryUtil.kt
/*
 * QueryUtil.java
 *
 * Copyright 2008 Jeff Campbell. All rights reserved. Unauthorized reproduction 
 * is a violation of applicable law. This material contains certain 
 * confidential or proprietary information and trade secrets of Jeff Campbell.
 */
package org.dbtools.query.shared.util

/**
 * @author Jeff
 */
object QueryUtil {
    fun formatLikeClause(column: String, value: String): String {
        return "$column LIKE $value"
    }

    fun formatIgnoreCaseLikeClause(column: String, value: String): String {
        return formatLikeClause(column, value)
    }
}
